packages/block-library/src/post-template/index.php

```diff
@@ -82,17 +82,28 @@ function render_block_core_post_template( $attributes, $content, $block ) {
 	$content = '';
 	while ( $query->have_posts() ) {
 		$query->the_post();
+
+		// Get an instance of the current Post Template block.
+		$block_instance = $block->parsed_block;
+
+		// Set the block name to one that does not correspond to an existing registered block.
+		// This ensures that for the inner instances of the Post Template block, we do not render any block supports.
+		$block_instance['blockName'] = 'core/null';
```

Comment:
What would be the option for developers who would want block supports for `core/post-template` later?

Render them somewhere in the render callback?

Reply:
The supports still get rendered on the outer wrapper (this function is called with `core/post-template`) so the block supports are still applied there.

I'm not too sure about how we'd handle styling for the inner wrapper to be honest — I think the best bet for pattern design would be to have a Group block as the first child of the inner wrapper, and then folks can use block supports there. I was _almost_ wondering if we could use a group block instead of `core/null`, but that felt like going down a rabbithole that might be a bit broader in scope than solving the current bug 🤔

Longer-term, I'd love for us to figure out how to reliably get block spacing and block supports working nicely for styling up different kinds of post template layouts, though!